eslint.config.mjs

```diff
@@ -11,6 +13,9 @@ export default tseslint.config(
   },
   eslint.configs.recommended,
   ...tseslint.configs.strictTypeChecked,
+  pluginImport.recommended,
+  // @ts-expect-error not sure why the plugin isn't type correctly, but it works...
```

Comment:
I removed this after including the fix mentioned above and did not see any errors.